exporter/prometheus/example_test.go

```diff
@@ -0,0 +1,37 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package prometheus_test
+
+import (
+	"log"
+	"net/http"
+
+	"go.opencensus.io/exporter/prometheus"
+	"go.opencensus.io/stats"
+)
+
+func Example() {
+	exporter, err := prometheus.NewExporter(prometheus.Options{})
+	if err != nil {
+		log.Fatal(err)
+	}
+	stats.RegisterExporter(exporter)
```

Comment:
defer stats.UnregisterExporter(exporter)

Also am not sure why we are putting this in a goroutine in this example, let's remove this, please.

Reply:
Ditto

Removed the goroutine.

Reply:
maybe add a comment saying something like "// register other handlers" because currently only the metrics handler is registered and then the program blocks forever which doesn't seem useful

Reply:
I think it is expected to start a new server for the diagnostics reasons and not expose them on your default server.